TFE: Pre-install: Detail Azure PostgreSQL extension allow-list (#2164)
hashicorp/terraform-website: content/enterprise/before-installing/data-storage-requirements/postgres-requirements.mdx

In Terraform Enterprise v202103-1, the internally-managed PostgreSQL server has been upgraded from PostgreSQL 9.5 to PostgreSQL 12. This change only affects mounted disk and proof of concept installations.

For more details, consult the v202103-1 [release notes](https://github.com/hashicorp/terraform-enterprise-release-notes/blob/master/v202103-1.md).

## Azure Database for PostgreSQL Flexible Server

> Azure requires that users of the Azure Database for PostgreSQL Flexible Server offering allow-list any desired PostgreSQL extensions for their PostgreSQL Flexible Server before they can be installed and/or used within a database.

Reviewer: ```suggestion
You must allow-list PostgreSQL extensions for your PostgreSQL Flexible Server before you can install them and use them within a database. To configure the Azure Database for PostgreSQL Flexible Server, specify the `CITEXT`, `HSTORE`, and `UUID-OSSP` extensions under the `azure.extensions` server parameters.
```

> If a manually configuring the Azure Database for PostgreSQL Flexible Server, simply ensure that the `CITEXT`, `HSTORE`, and `UUID-OSSP` extensions are checked under the `azure.extensions` server parameters. Alternatively, if Terraform is used to manage the database, this example resource should appropriately add the extensions Terraform Enterprise utilizes to the allow-list:

Reviewer: you had originally said "Check" the extensions - is this something you'd do in the UI? otherwise, I'm wondering if we should use a different verb. 

```suggestion
To use Terraform to manage the database, add the extensions that Terraform Enterprise will use to the allow-list. The following example resource adds the extensions for Terraform Enterprise to the allow-list. 
```
